templates/config.json.j2

```diff
@@ -0,0 +1,200 @@
+{
+	"version": "33",
+	"credential": {
+		"accessKey": "{{ minio_access_key }}",
+		"secretKey": "{{ minio_secret_key }}",
+		"expiration": "1970-01-01T00:00:00Z",
```

Comment:
???

Reply:
This is the credentials for accessing Minio. And in this case we install it ourselves.
Expiration is left in the default state. I understand that if this date does not come, then the expiration date will also not come ))